spec/js/ko-components/views/orifice-calculator-spec.coffee

```diff
@@ -16,27 +15,57 @@ describe 'orifice-calculator-view-spec', ->
     it 'should be a dropdown list', ->
       expect($(element)).toEqual 'SELECT'
 
+  itShouldBehaveLikeTextInput = () ->
```

Comment:
Same theory of passing an argument to the function instead of a global variable applies here